### PR TITLE
Make the Header Logo link on OQRS page go back to public slug page

### PR DESCRIPTION
**Pull request by phl0**
Initially set up to make the user go back to public sluc page from OQRS page.
Ended up cleaning visitor's header and footer from CI calls and assigned the variables in corresponding controllers.

### CODE REVIEW

File: application/controllers/Oqrs.php
Comment (HB9HIL) [inline application/controllers/Oqrs.php:34]: we should not pass `$data` twice to a page


```suggestion
		$this->load->view('interface_assets/footer');
```

**Review by HB9HIL — CHANGES_REQUESTED** [review]
can't spot any other issues in this PR :) good job!
after removing `$data` mentioned in the comment this one is good for me.

**Review by HB9HIL — APPROVED** [review]
looks good to me :) let's wait for @AndreasK79 approval aswell

**Review by AndreasK79 — APPROVED** [review]

**Comment by AndreasK79** [comment]
Works while testing. Also removed the index in the url made it nicer. And code is even simpler now with the dynamic mapping function.

**Pull request merged**
